Admin Actions Updates (#193)
pulsar-edit/package-backend: docs/reference/Admin_Actions.md

With that said this document will serve as the ongoing history of administrative actions that must be taken against the backend.

## 2023 - August 16

### appcelerator-titanium && titanium

> When the developers of `appcelerator-titanium` wanted to rebrand their package to `titanium` it seemed the backend was encountering errors attempting to do so. The developers then went ahead to create a brand new package `titanium` and continue development there. Unfortunately, also encountering errors when attempting to delete the now defunk package `appcelerator-titanium`. The developers had then [reached out](https://github.com/pulsar-edit/package-backend/issues/190) to the Pulsar Backend team for assistance.

Reviewer: ```suggestion
When the developers of `appcelerator-titanium` wanted to rebrand their package to `titanium` it seemed the backend was encountering errors attempting to do so. The developers then went ahead to create a brand new package `titanium` and continue development there. Unfortunately, also encountering errors when attempting to delete the now defunct package `appcelerator-titanium`. The developers had then [reached out](https://github.com/pulsar-edit/package-backend/issues/190) to the Pulsar Backend team for assistance.
```

> Since none of this issues where the fault of the developers and seem to fully fall on the backend itself, the request was happy to be met, which attempted to do the following:

Reviewer: ```suggestion
Since none of these issues were the fault of the developers and seem to fully fall on the backend itself, the request was happy to be met, which attempted to do the following:
```

* Remove `appcelerator-titanium`
* Transfer what data possible over to `titanium`

To keep with this request, the following actions have been made:

* `appcelerator-titanium` has been deleted from the PPR, leaving the name reserved to avoid Supply Chain Attacks
* `appcelerator-titanium`s 6 `original_stargazers` (That is unindexed stargazers transferred from Atom) have been added to `titanium`s 0 `original_stargazers`
* `appcelerator-titanium`s 11,652 `downloads` have been added to `titanium`s 114 `downloads`. Resulting in a new value of `11,766` downloads for the `titanium` package.

### atom-oss-license

The community package [`atom-oss-license`](https://web.pulsar-edit.dev/packages/atom-oss-license) ([repo](https://github.com/mmk2410/atom-oss-license)), has been archived on March 9, 2019.

> In an effort to make information like this more easily available to users of Pulsar, we do intend to mark any packages that originate from an archived forge.

Reviewer: ```suggestion
In an effort to make information like this more easily available to users of Pulsar, we do intend to mark any packages that originate from an archived repo.
```
Should this be repo rather than forge?
